tests/run/pyclass_annotations_pep526.py

```diff
@@ -0,0 +1,45 @@
+# cython: language_level=3
+# mode: run
+# tag: pure3.7, pep526, pep484
+
+from __future__ import annotations
+
+try:
+    from typing import ClassVar
```

Comment:
I just meant this line, not the whole class. It should work without the import, shouldn't it?

I just noticed that the other test file that you changed does this:
```python
from typing import Dict, List, TypeVar, Optional, Generic, Tuple
try:
    from typing import ClassVar
except ImportError:
    ClassVar = Optional  # fake it in Py3.5
```
That should work here as well.

Reply:
Done.